test/fail_compilation/fail3.d

```diff
@@ -14,21 +14,14 @@ template vector(T)
         T x, y;
     }
 
+    // not struct member
     vec2 opAdd(vec2 a, vec2 b)
     {
         vec2 r;
         r.x = a.x + b.x;
         r.y = a.y + b.y;
         return r;
     }
-
-    vec2 make2(T x, T y)
-    {
-        vec2 a;
-        a.x = x;
-        a.y = y;
-        return a;
-    }
```

Comment:
Why was this removed?

Reply:
It doesn't seem relevant to the test - should I restore it?

Reply:
Yes, it does seem unnecessary, but I don't think it belongs in this PR.

Reply:
OK, restored.